Extensions/editable_reblogs.js

```diff
@@ -341,6 +346,7 @@ XKit.extensions.editable_reblogs = new Object({
 
 	make_post: function(e) {
 		if (!this.reblog_tree_exists() || this.state != "success") {
+			XKit.interface.post_window.set_content_html(this.wrap_html_links(XKit.interface.post_window.get_content_html()));
```

Comment:
This feels like a race condition—how do we know whether our event handler or tumblr's event handler will be called first?